### PR TITLE
Ipanema mixed mode

### PR DESCRIPTION
**Pull request by hellt**
This lab demonstrates a joint integration between Nuage Networks VNS and Infovista Ipanema solutions. The difference this lab has from the `0800-VNS-IPANEMA_WAN_OPT` lab, is that it demonstrates how the Nuage Networks VNS can integrate with Infovista Ipanema, where existing sites are already using Ipanema Physical Appliances, hence the mixed deployment model name with virtual and physical Ipanema appliances.

### CODE REVIEW

**Comment by hellt** [comment]
do not merge yet, I am considering adding some more installation instructions to add

File: 0801-VNS-IPANEMA_WAN_OPT_MIXED_MODE/README.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+Once a lab is deployed and configured using automation scripts, it will match the following diagram:
+![lab](https://www.lucidchart.com/publicSegments/view/5b512e4d-9075-4cfc-8ea5-03805917dde1/image.png)
+
+The lab utilizes a single "Internet" underlay and a demo Organization with a HQ site and three branches, where the headquarter site uses Ipanema Physical Engine (IPE) and all of the other branches support Virtual Ipanema Engines (VIPEs).
+
+The headquarters site user devices and branch offices user devices are emulated using the [Branch-PC image](https://nuagenetworks.zendesk.com/hc/en-us/articles/360010244033) which allows us to generate and analyze traffic as well as run some real-world applications.
+
+# Deployment
+The lab deployment process is powered by the [nuxctl](https://nuxctl.nuagex.io) CLI tool. All of the infrastructure deployment will be completed after a user runs the the tool against the [lab template](nuxctl_0801-vns-ipanema_wan_opt_mixed_mode.yml) supplied within this repo.
+
```
Comment (apooniajjn) [inline 0801-VNS-IPANEMA_WAN_OPT_MIXED_MODE/README.md:31]: Can you tell end user to install nuxctl first before proceeding to lab deployment steps

Comment (hellt) [reply 0801-VNS-IPANEMA_WAN_OPT_MIXED_MODE/README.md:31]: yes, absolutely valid. I have it added for viavi lab and I plan to add it to this PR as well

**Comment by hellt** [comment]
@hkhazaal @apooniajjn 
I have removed WIP status and augmented the README file with more details on the prerequisites. 

You can check the changes I made from the last commit https://github.com/nuagex/nuagex-labs/pull/7/commits/5353c6e4b13dab6114363c58887dc0fd1b232520

I also tested the lab deployment using the `CATS in a container` approach and validated it.

My next plan is to change the `ipanema hosted vnf lab` to containerized CATS and then the Ipanema automation artefacts could be considered complete

**Pull request merged**